pkg/flypg/zombie_test.go

```diff
@@ -0,0 +1,243 @@
+package flypg
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+)
+
+func TestZombieDiagnosis(t *testing.T) {
```

Comment:
this is excellent. this makes it easier for me to continue to threaten to make real integration tests :)